src/main/java/com/joss/conductor/mobile/annotation/AndroidId.java

```diff
@@ -0,0 +1,17 @@
+package com.joss.conductor.mobile.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Created on 3/14/18.
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface AndroidId {
+
+    String value() default "";
```

Comment:
Would it make sense to update this to `id` if we make the above change?

Reply:
`value()` is a reserved word for custom annotations that allows the shorthand  `@AndroidFindBy("button")` vs. `@AndroidFindBy(id = "button")`.  I was assuming IDs would be the main use case and wanted to reduce typing out `id = ` over and over.  If someone only wanted to specify`xpath = ` it would need to be written out.  
https://docs.oracle.com/javase/specs/jls/se9/html/jls-9.html#jls-9.7.3

Reply:
And `value()` is just temporary until it gets set in the helper either `By.id`(Android) or `By.name`(iOS - will implement this change from your comment below).  So I would opt for leaving it as `value()` in the annotation and let people use the shorthand.  I could add a comment here to let users know what the value() gets set to later on.